### PR TITLE
Fix mavlink2rest bugs

### LINKED ISSUE #5: Bugs in mavlink2resthelper
Discovered two bugs in the mavlink2resthelper.py file:

1. Import in line 1 references companionhelper, should be blueoshelper.

2. The GET requests in line 175 and 186 are failing due to incomplete paths, likely due to BlueOS using a newer mavlink2rest version. Can be fixed by adding the constant:
`PATH_PREFIX = "/vehicles/1/components/1/messages"`

And updating the aforementioned lines to:
`response = request(MAVLINK2REST_URL + '/mavlink' + PATH_PREFIX + path)`

Ideally the path prefix should allow for accessing different vehicle and components IDs, unsure how to best implement this.

### PR DESCRIPTION
**Pull request by ES-Alexander**
- Fixes #5 
- Vehicle and component specifiers aren't properly accessible - need some rework of `dvl.py`, `main.py`, and front end
- May also be worth incrementing version number - not sure if it's best to use `1.0.1` or `1.1.0`

### CODE REVIEW

File: dvl-a50/mavlink2resthelper.py [hunk 1]
```diff
@@ -166,24 +169,29 @@ def __init__(self):
 }}
 """
 
-    def get_float(self, path: str) -> float:
+    def get_float(self, path: str, vehicle: int=None, component: int=None) -> float:
```
Comment (patrickelectric) [inline dvl-a50/mavlink2resthelper.py:172]: The type here is wrong, should not be int, but Optional[int]

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: dvl-a50/mavlink2resthelper.py (hunk 1 above)
Comment (ES-Alexander) [reply dvl-a50/mavlink2resthelper.py:172]: Should we also change the return signature of `get` to be something that allows float or bool?

Comment (patrickelectric) [reply dvl-a50/mavlink2resthelper.py:172]: We should choose between returning an optional over an float/nan.
Bool does not make a lot of sense to me.

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: dvl-a50/mavlink2resthelper.py (hunk 1 above)
Comment (ES-Alexander) [reply dvl-a50/mavlink2resthelper.py:172]: Sorry, I meant `str` or `bool`, since that's what it actually returns (this is for `get` - not `get_float`).

**Pull request closed without merging**